src/core/core.ticks.js

```diff
@@ -65,15 +65,8 @@ module.exports = {
 			return tickString;
 		},
 
-		logarithmic: function(tickValue, index, ticks) {
-			var remain = tickValue / (Math.pow(10, Math.floor(math.log10(tickValue))));
-
-			if (tickValue === 0) {
-				return '0';
-			} else if (remain === 1 || remain === 2 || remain === 5 || index === 0 || index === ticks.length - 1) {
-				return tickValue.toExponential();
-			}
-			return '';
+		logarithmic: function(tickValue) {
+			return tickValue === 0 ? '0' : tickValue.toExponential();
```

Comment:
So I understand correctly, this is safe because we only generate ticks for the 1,2,5, 10 cases?

Reply:
Not quite. We don't generate ticks only for the 1, 2, 5, 10 cases. That was done before to keep only nice looking ticks. The ticks that are generated now would be 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, etc. Then we mark 1, 10, 100, etc. as major ticks and let the autoSkipper figure out if we need to keep any in between. So for example, now we could end up with 1, 5, 10, 50, 100. We can have more or less ticks depending on how big the scale is, what `autoSkipPadding` is set to, etc. which I think gives the user more control